csv.py

```diff
@@ -0,0 +1,27 @@
+import json                                                     
+import os
+from flask import Flask, request
+from werkzeug import secure_filename
+import panda as pd
+import numpy as np
+
+app = Flask(__name__)                                           
+@app.route('/')    
+def upload_file():
+  uploaded_file = request.files['document']
+  # data = json.load(request.files['data'])
+  filename = secure_filename(uploaded_file.filename)
+  uploaded_file.save(os.path.join('/', filename))
```

Comment:
I believe we should save the file at the CWD (current working directory) of the server and not at `/` the root directory.

`filename` will not be an absolute path but just the name of the file.